packages/react/src/themes/teams/components/Button/buttonVariables.ts

```diff
@@ -82,45 +76,39 @@ export default (siteVars: any): ButtonVariables => ({
   contentFontWeight: siteVars.fontWeightSemibold,
   contentLineHeight: siteVars.lineHeightMedium,
 
-  color: siteVars.colors.grey[750],
-  colorHover: siteVars.colors.grey[750],
-  colorFocus: siteVars.colors.grey[750],
+  color: siteVars.colorScheme.default.foreground,
+  colorHover: siteVars.colorScheme.default.foregroundHover,
+  colorFocus: siteVars.colorScheme.default.foreground,
+  colorActive: siteVars.colorScheme.default.foregroundPressed,
   colorDisabled: siteVars.colorScheme.brand.foregroundDisabled,
-  backgroundColor: siteVars.colors.white,
-  backgroundColorActive: siteVars.colors.grey[200],
-  backgroundColorHover: siteVars.colors.grey[50],
-  backgroundColorFocus: siteVars.colors.grey[200],
-  backgroundColorDisabled: siteVars.colors.grey[150],
-  borderColor: siteVars.colors.grey[200],
-  borderColorHover: siteVars.colors.grey[250],
+  backgroundColor: siteVars.colorScheme.default.background,
+  backgroundColorActive: siteVars.colorScheme.default.backgroundPressed,
+  backgroundColorHover: siteVars.colorScheme.default.backgroundHover1,
+  backgroundColorFocus: '',
+  backgroundColorDisabled: siteVars.colorScheme.default.backgroundDisabled1,
+  borderColor: siteVars.colorScheme.default.border,
+  borderColorHover: siteVars.colorScheme.default.borderHover,
+  borderColorActive: siteVars.colorScheme.default.borderPressed,
   borderColorDisabled: 'transparent',
 
-  primaryColor: siteVars.colors.white,
-  primaryColorHover: siteVars.colors.white,
-  primaryColorFocus: siteVars.colors.white,
-  primaryBackgroundColor: siteVars.colors.brand[600],
-  primaryBackgroundColorActive: siteVars.colors.brand[900],
-  primaryBackgroundColorHover: siteVars.colors.brand[800],
-  primaryBackgroundColorFocus: siteVars.colors.brand[800],
+  primaryColor: siteVars.colorScheme.brand.foreground4,
+  primaryColorHover: siteVars.colorScheme.brand.foreground4,
+  primaryBackgroundColor: siteVars.colorScheme.brand.background,
+  primaryBackgroundColorActive: siteVars.colorScheme.brand.backgroundPressed,
+  primaryBackgroundColorHover: siteVars.colorScheme.brand.backgroundHover,
+  primaryBackgroundColorFocus: '',
```

Comment:
should this be the same as primaryBackgroundColor now? Can we replace the `''` with some value or remove the variable?